python/cudf/cudf/tests/test_orc.py

```diff
@@ -301,27 +301,26 @@ def test_orc_read_rows(datadir, skiprows, num_rows):
     assert_eq(pdf, gdf)
 
 
-def test_orc_read_skiprows(tmpdir):
+def test_orc_read_skiprows():
     buff = BytesIO()
-    df = pd.DataFrame(
-        {"a": [1, 0, 1, 0, None, 1, 1, 1, 0, None, 0, 0, 1, 1, 1, 1]},
-        dtype=pd.BooleanDtype(),
-    )
+    data = [
+        True,
+        None,
+        True,
+        False,
+        None,
+        True,
+        True,
+        False,
+    ]
     writer = pyorc.Writer(buff, pyorc.Struct(a=pyorc.Boolean()))
-    tuples = list(
-        map(
-            lambda x: (None,) if x[0] is pd.NA else (bool(x[0]),),
-            list(df.itertuples(index=False, name=None)),
-        )
-    )
-    writer.writerows(tuples)
+    writer.writerows([(d,) for d in data])
     writer.close()
 
-    skiprows = 10
+    skiprows = 3
```

Comment:
Can we retain the same shape and skiprows count since the previous one's were not quite big? AFAIK this was introduced for boolean specific bug fix and didn't repro for other sizes of data

Reply:
Done